src/Microsoft.Management.Infrastructure.CimCmdlets/Utils.cs

```diff
@@ -203,7 +203,7 @@ internal static bool GenrateVerboseMessage
         internal static string removeParameterSetName = @"Remove parameter set {0} name from cache";
         internal static string currentParameterSetNameCount = @"Cache have {0} parameter set names";
         internal static string currentParameterSetNameInCache = @"Cache have parameter set {0} valid {1}";
-        internal static string currentnonMadatoryParameterSetInCache = @"Cache have optional parameter set {0} valid {1}";
+        internal static string currentnonMandatoryParameterSetInCache = @"Cache have optional parameter set {0} valid {1}";
```

Comment:
I don't see any usages of any of these internal strings. @lzybkr We should investigate and consider removing them.